src/StatusQ/Controls/StatusIdenticonRing.qml

```diff
@@ -0,0 +1,60 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.0
+
+import StatusQ.Core 0.1
+
+Item {
+    id: identiconRing
+
+    property StatusIdenticonRingSettings settings: StatusIdenticonRingSettings {
+        ringSpec: undefined
+        totalRingUnits: 1
+        initalAngleRad: 0
+        ringPxSize: 1.5
+    }
+
+    readonly property real segmentRadLen: 2 * Math.PI / settings.totalRingUnits
+    // 32 distinctive segment: They are expected to be used using the array index from 0 to 31 and with the order defined in figma (it should be defined as well in specs.)
+    readonly property var distinctiveColors: ["#000000", "#726F6F", "#C4C4C4", "#E7E7E7", "#FFFFFF", "#00FF00",
```

Comment:
I would NOT make it read-only. Clients of StatusQ may want to define their own colors.